packages/server-core/src/setting/engine-setting/engine-setting.ts

```diff
@@ -64,6 +64,7 @@ export default (app: Application): void => {
       'task-server': appConfig.taskserver,
       coil: appConfig.coil,
       chargebee: appConfig.chargebee,
+      'instance-server': appConfig.instanceserver,
```

Comment:
Have you tested, how `onUpdateAppConfig` works when webrtcsettings are changed?

Reply:
when webRTCSettings is changed it trigger onUpdateAppConfig

but in the appconfig the webRTCSettings is not setup 

```  const instanceserver = {
  clientHost: process.env.APP_HOST!,
  rtcStartPrt: parseInt(process.env.RTC_START_PORT!),
  rtcEndPort: parseInt(process.env.RTC_END_PORT!),
  rtcPortBlockSize: parseInt(process.env.RTC_PORT_BLOCK_SIZE!),
  identifierDigits: 5,
  local: process.env.LOCAL === 'true',
  domain: process.env.INSTANCESERVER_DOMAIN || 'instanceserver.etherealengine.com',
  releaseName: process.env.RELEASE_NAME || 'local',
  port: process.env.INSTANCESERVER_PORT!,
  locationName: process.env.PRELOAD_LOCATION_NAME!,
  shutdownDelayMs: parseInt(process.env.INSTANCESERVER_SHUTDOWN_DELAY_MS!) || 0,
}

Reply:
@hanzlamateen I just added WebRTCSettings appConfig.instanceServer and tested it